services/src/main/java/org/broadleafcommerce/payment/service/gateway/DefaultPayPalCheckoutTransactionService.java

```diff
@@ -211,9 +226,13 @@ public PaymentResponse authorizeAndCapture(PaymentRequest paymentRequest) {
                         payerLastName = payer.getPayerInfo().getLastName();
                     }
 
+                    String updateTime = payment.getUpdateTime();
+
                     paymentResponse
```

Comment:
For the AuthorizeAndCapture interaction, can we confirm that the `transactionReferenceId` is being effectively passed to the gateway?